theano/tensor/extra_ops.py

```diff
@@ -11,6 +11,43 @@
 tensor = basic
 
 
+class cpu_contiguous(theano.Op):
```

Comment:
Camel case name.

Remove eq and hash methods. Add the `__props__ = ()` attribute.

Set an attribute destroy_map = {0:0}